client/jetpack-cloud/sections/sidebar-navigation/sites.tsx

```diff
@@ -20,7 +20,10 @@ const SitesSidebar = ( { path }: { path: string } ) => {
 		...props,
 		path: JETPACK_MANAGE_SITES_LINK_NEEDS_ATTENTION,
 		trackEventName: 'calypso_jetpack_sidebar_menu_click',
-		isSelected: props.link === path,
+		isSelected:
+			props.link === JETPACK_MANAGE_SITES_LINK_FAVORITES
+				? props.link === path.split( '?' )[ 0 ]
+				: props.link === path.split( '&' )[ 0 ],
```

Comment:
Maybe this is what you want to do here?

```
const isSelected = props.link === JETPACK_MANAGE_SITES_LINK_FAVORITES
    || props.link === path.split('?')[0]
    || props.link === path.split('&')[0];
```

Using a chain of `OR`.

Reply:
Or maybe:

`isSelected: props.link.startsWith( JETPACK_MANAGE_SITES_LINK_FAVORITES );`

Reply:
That breaks the site selection in the sidebar unfortunately. 

However, I can remove the changes here entirely as favorites has now been removed as a filter option and there are no new params added to the 'all_issues' filter URL, so the existing implementation works fine.

Reply:
59c03b2766a0b680f14615e04ecd0d01a677047